CHANGES.md

```diff
@@ -1,4 +1,8 @@
-# UNRELEASED
+# 2.0.0 (UNRELEASED)
+- [NEW] Added API for upcoming IBM Cloud Identity and Access Management support
+  for Cloudant on IBM Cloud. Note: IAM API key support is not yet enabled in the
+  service.
+- [NEW] Support multiple plugins.
```

Comment:
Should we have a link to the documentation? I understand that the upgrade implications are minimal and that existing use of one plugin will continue to work seamlessly, is that right and should that be made more obvious?